packages/documentation/package.json

```diff
@@ -1,19 +1,22 @@
 {
-  "name": "docusaurus",
+  "name": "documentation",
```

Comment:
Even though this package won't be published to npm, just for consistency with the other packages, I think this should be:

```suggestion
  "name": "@spaship/documentation",
```

Reply:
Great idea! :+1: